src/workerd/api/actor-state.h

```diff
@@ -536,6 +543,7 @@ class DurableObjectState: public jsg::Object {
     JSG_METHOD(waitUntil);
     JSG_READONLY_INSTANCE_PROPERTY(id, getId);
     JSG_READONLY_INSTANCE_PROPERTY(storage, getStorage);
+    JSG_READONLY_INSTANCE_PROPERTY(container, getContainer);
```

Comment:
Nit: these could probably be `JSG_LAZY_READONLY_INSTANCE_PROPERTY`

Reply:
That does seem like a win, since none of these can actually change value.

Additionally, there's probably no reason for them to be read-only. In fact, making the new member read-only is arguably a breaking change. So I'm adding a commit to make all three be `JSG_LAZY_INSTANCE_PROPERTY`, for consistency.

Reply:
263525eed3a89a2e5ffbce20a9b6f558c217404f